### PR TITLE
Add LICENSE file.

### PR DESCRIPTION
**Pull request by kcwu**
LICENSE file in the top directory is required according to
http://www.apache.org/dev/apply-license.html

### CODE REVIEW

**Comment by odeke-em** [comment]
Thanks @kcwu for looking out for this. However we already have a license reference in the README file.
screen shot 2015-03-20 at 12 39 33 pm

**Comment by kcwu** [comment]
I know. It's still better to have LICENSE file with the code.

**Comment by odeke-em** [comment]
But these are just alternating styles. Including the short license in the README to me is sufficient instead of having an explicit LICENSE file. Also every file containing source code begins with the LICENSE.

**Comment by odeke-em** [comment]
@kcwu okay how about you use the short version of the LICENSE as is in the README? Then I can merge it in.

**Comment by kcwu** [comment]
I think if it is "LICENSE" file, it should be full copy of the license text.

**Comment by odeke-em** [comment]
Aye aye. At least fill in the details of the copyright holder as "Google Inc" and year as 2013.

**Comment by kcwu** [comment]
I don't understand what you mean. Which paragraph are you talking about?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (odeke-em) [inline LICENSE]: Right here, put the copyright holder and year.

**Comment by kcwu** [comment]
That paragraph is teaching how to apply Apache license. If [yyyy] [name of copyright owner] are substituted, users may be confusing what does this sentence mean
```
To apply the Apache License to your work, attach the following
boilerplate notice, with the fields enclosed by brackets "[]"
replaced with your own identifying information. (Don't include
the brackets!) 
```

**Comment by odeke-em** [comment]
Never mind, I'll clean up. Thank you for the PR!

**Pull request merged**